src/Database/PG/Query/PTI.hs

```diff
@@ -1,161 +1,484 @@
-{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
-
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE PatternSynonyms #-}
+
+-- |
+-- Description: Oid constants.
+--
+-- All these are taken from this declaration file in the postgres repo:
+-- https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_type.dat
 module Database.PG.Query.PTI where
 
--------------------------------------------------------------------------------
+import Database.PostgreSQL.LibPQ (Oid (..))
+
+-- | Convert an 'Oid' for a single type into the corresponding array 'Oid'. If
+-- the original Oid is 'Auto', then the result will also be 'Auto'. For types
+-- without a defined array parallel, the result will be 'Unknown'.
+arrayOf :: Oid -> Oid
+arrayOf = \case
+  Auto -> Auto
+  ACLItem -> ACLItemArray
```

Comment:
Just realised I committed the capitalisation fixes to the wrong branch... >_> @nicuveo

Reply:
https://github.com/hasura/pg-client-hs/pull/70/files#diff-6a999c381a106e01601f6102eaaa8212bf3545be7fc8bdc0a9d438b035f61ddeR16 or not...

Reply:
Merge conflicts, never mind 😅